aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -108,6 +109,10 @@ module aptos_framework::block {
         // transition is the last block in the previous epoch.
         stake::update_performance_statistics(proposer_index_optional, failed_proposer_indices);
 
+        if (round == 0) {
```

Comment:
no this doesn't work, we don't have round 0 block. it's even worse, the first round in the new epoch can be different

Reply:
So how?

Reply:
well you need to maintain something yourself that reset at end of the epoch

Reply:
see the new commit

Reply:
lgtm